src/inmanta/server/services/metricservice.py

```diff
@@ -34,6 +38,7 @@ def __init__(self) -> None:
         self._influx_db_reporter: Optional[InfluxReporter] = None
 
     async def start(self) -> None:
+        self.start_auto_benchmark()
```

Comment:
I got confused here. The method name says `start_auto_benchmark()`, but as far as I can see it only registers the `CPUMicroBenchMark` to the global registry. I think we need to rename this method.

Reply:
We may add more in the future, I think.